### ISSUE TITLE
Short lib name

### ISSUE THREAD

**Issue by didierfranc** [elisherer/react-waterfall-redux-devtools-middleware#3]
There is some available name that could be better, I think

```
react-waterfall-devtools
react-waterfall-redux
```

or something like that

**Comment by elisherer** [comment]
I see you changed v4 to include it anyway... do you still want this to exist?

**Comment by didierfranc** [comment]
Yes what do you think of that ? https://github.com/didierfranc/react-waterfall/blob/master/src/helpers/devtools.js

It implement only time travel

**Comment by elisherer** [comment]
Looks very cool, and it looks you are using an API,
I didn't know they had one. I used postMessage and it was very hard.

BTW: They changed the name to `window.__REDUX_DEVTOOLS_EXTENSION__` the old one is deprecated.
I think I will drop this lib in favor of your embedded code.

**Comment by fkerekiglobant** [comment]
How would you use this with React Native, is it possible?
